corehq/messaging/smsbackends/airtel_tcl/tests/test_airtel_tcl.py

```diff
@@ -110,14 +110,14 @@ def test_get_url(self):
         )
 
     def test_get_phone_number(self):
-        self.assertEqual(AirtelTCLBackend.get_phone_number('+910123456789'), '0123456789')
-        self.assertEqual(AirtelTCLBackend.get_phone_number('910123456789'), '0123456789')
+        self.assertEqual(AirtelTCLBackend.get_phone_number('+918123456789'), '8123456789')
+        self.assertEqual(AirtelTCLBackend.get_phone_number('916123456789'), '6123456789')
```

Comment:
just dropping this, please ignore if you are already on it, we would need a new test for number with correct length but not starting with digits we expect.